<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParserItem</class>
 <widget class="QWidget" name="ParserItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>125</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>125</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>125</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>OpenALPR Hotlist Importer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>List &amp;Name</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="buddy">
                 <cstring>txtName</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtName">
                <property name="toolTip">
                 <string>The name of the alert list that will be pushed to the OpenALPR Web Server</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="btnHelpName">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
			<iconset theme=":resources/help-icon.ico">
				<normaloff>:resources/help-icon.ico</normaloff>:resources/help-icon.ico
                  </iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>22</width>
                  <height>22</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Parse &amp;Code</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="buddy">
                 <cstring>txtCode</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtCode">
                <property name="toolTip">
                 <string>A special keyword that the hotlist parser uses to identify this alert list type in the input hotlist data.</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="btnHelpCode">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
			<iconset theme=":resources/help-icon.ico">
				<normaloff>:resources/help-icon.ico</normaloff>:resources/help-icon.ico
		 </iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>22</width>
                  <height>22</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Match &amp;Strategy</string>
                </property>
                <property name="buddy">
                 <cstring>comboMatchStrategy</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboMatchStrategy">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>90</width>
                  <height>16777215</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>Exact</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Lenient</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>100</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="chkOverride">
                <property name="toolTip">
                 <string>Check to use a different hotlist file than the one defined in the general &quot;Hotlist Location&quot;</string>
                </property>
                <property name="text">
                 <string>Override</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="btnHelpOverride">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
			<iconset theme=":resources/help-icon.ico">
				<normaloff>:resources/help-icon.ico</normaloff>:resources/help-icon.ico
                  </iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>22</width>
                  <height>22</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Hotlist &amp;Location</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="buddy">
                 <cstring>txtCode</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtHotlistLocation">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="layoutButtons">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnRemove">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtCode</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>txtName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ParserItem</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboMatchStrategy</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ParserItem</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtCode</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ParserItem</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>989</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtHotlistLocation</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ParserItem</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>changed(QString)</signal>
  <slot>changed()</slot>
 </slots>
</ui>
